docs/sphinx/_toc.yml.in

```diff
@@ -3,29 +3,35 @@
 root: index
 subtrees:
   - entries:
-      - file: what-is-migraphx
-      - file: migraphx-driver
-        subtrees:
-          - entries:
-            - file: reference/driver-options
-      - file: dev/contributing-to-migraphx
-        subtrees:
-          - entries:
-            - file: dev/data
-            - file: dev/operators
-            - file: dev/program
-            - file: dev/targets
-            - file: dev/quantization
-            - file: dev/pass
-            - file: dev/matchers
-            - file: dev/tools
-            - file: dev/env_vars
-            
-  - caption: API Reference
+    - file: what-is-migraphx
+    
+  - caption: API reference
     entries:
     - file: reference/cpp
     - file: reference/py
 
+  - caption: Command-line tool
+    entries:
+    - file: migraphx-driver
```

Comment:
This should go under the reference section. It shouldn't be a seperate section.

Reply:
@pfultz2 https://advanced-micro-devices-demo--2884.com.readthedocs.build/projects/AMDMIGraphX/en/2884/ Let me know if it looks good to you now.